contracts/modules/Affiliates.sol

```diff
@@ -181,6 +181,7 @@ contract Affiliates is State, AffiliatesEvents {
 		if (bonusPaymentIsSuccess) {
 			emit PayTradingFeeToAffiliate(
 				referrer,
+				tx.origin, // trader
```

Comment:
**potential issue**
this can break under certain conditions
theoretically you can borrow funds or open a position for a different wallet than the one you're sending with
this could be important e.g. for contracts which means that tx.origin != trader
another point is that initially the LoanToken function marginTrade and then FeesHelper function _payTradingFee accepts user as a parameter which is passed down the function calls
we should not break trough the interface boundaries and use inferred logic that as it can change at some point

**proposed solution**
add trader as a parameter to the functions when needed